src/main/java/com/example/bean/Player.java

```diff
@@ -8,9 +8,10 @@
 
     private Score scorePlayer;
 
-    private short codePlayer;
+    private Score codePlayer;
+    //private short codePlayer;
 
```

Comment:
When I use `short` form in Player class, I got the following error.
If I changed it to score form, It run correctly.
Now, I put score form, so I might have to Undo.
![Screen Shot 2020-11-22 at 20 04 49](https://user-images.githubusercontent.com/16487285/99930885-88519100-2d07-11eb-823a-7a7f2ce43e8b.png)